src/InProcessTestHost/Sidecar/Dispatcher/TaskOrchestrationDispatcher.cs

```diff
@@ -166,12 +232,24 @@ static int GetSortOrderWithinGroup(TaskMessage msg)
         }
     }
 
+    static string GetShortHistoryEventDescription(HistoryEvent e)
+    {
+        if (Utils.TryGetTaskScheduledId(e, out int taskScheduledId))
+        {
+            return $"{e.EventType}#{taskScheduledId}";
+        }
+        else
+        {
+            return e.EventType.ToString();
+        }
+    }
+
     void ApplyOrchestratorActions(
         OrchestratorExecutionResult result,
         ref OrchestrationRuntimeState runtimeState,
-        out IList<TaskMessage> activityMessages,
-        out IList<TaskMessage> orchestratorMessages,
-        out IList<TaskMessage> timerMessages,
+        out List<TaskMessage> activityMessages, // CA1859: Use concrete types for better performance
```

Comment:
Nit, was this warning there before just not suppressed? Should we have a TODO to come back and fix it or does it not really matter since this is a test platform and that's talking about perf?